src/components/Analytics.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable */
+import React from 'react'
+
+class Analytics extends React.Component {
```

Comment:
Does this apply to both new components or just the analytics one? Here are some reasons why I like this approach:

1. These things change the DOM so it makes sense to have them be components, which change the DOM
2. We get syntax highlighting since its actual javascript instead of strings of javascript code. This might not matter as much right now, but I know there is upcoming DAP integration work where having syntax highlighting will be useful.